odh-dashboard/base/odh-dashboard-crd.yaml

```diff
@@ -89,26 +89,13 @@ spec:
                       type: string
                     pvcSize:
                       type: string
+                    gpuSetting:
+                      description: Configure how the GPU field works on the Jupyter tile. One of 'autodetect' (default, fetches for information), 'hidden' (remove the field) or a number-string (eg '5') to specify a hardcoded 0 to that number options
+                      type: string
                     notebookTolerationSettings:
                       type: object
                       properties:
-                          enabled: 
+                          enabled:
                             type: boolean
-                          key: 
+                          key:
                             type: string
-            status:
```

Comment:
can someone confirm for me this will be ok to upgrade on a cluster with an old status field?  I'm fine if status gets deleted, just want to make sure nothing unexpected happens.

Reply:
Reverting that part back, it's not in upstream, but i think this is rhods specific, as im not seeing any deletion in the history of the file in odh.

Reply:
ok, let's test it out and get them synced up in a less urgent fix.  approving...

/lgtm